gitman/cli.py

```diff
@@ -300,6 +300,9 @@ def _run_command(function, args, kwargs):
     except exceptions.ScriptFailure as exception:
         _show_error(exception)
         exit_message = "Run again with '--force' to ignore script errors"
+    except exceptions.InvalidConfig as exception:
+        _show_error(exception)
+        exit_message = "Adapt config and run again"
```

Comment:
I think we can probably leave `exit_message` blank here if the error is clearly shown.

Reply:
The error message is for example: 
  > Name conflict detected between source name and group name "gitman1".
I think for a newcomer it is helpful to know that an adaption of the config is needed.